src/main/obsRecorder.ts

```diff
@@ -158,44 +160,45 @@ const setupScene = (monitorIndex: number) => {
   return scene;
 }
 
-/*
-* getAudioDevices
-*/
-const getAudioDevices = (type: any, subtype: any) => {
-  const dummyDevice = osn.InputFactory.create(type, subtype, { device_id: 'does_not_exist' });
-  const devices = dummyDevice.properties.get('device_id').details.items.map(({ name, value }) => {
-    return { device_id: value, name,};
-  });
-  dummyDevice.release();
-  return devices;
-};
-
 /*
 * setupSources
 */
 const setupSources = (scene: any) => {
+  const cfg = new Store();
+  const selectedInputAudioDevice = (cfg.get('audio-input-device', 'all') as string);
```

Comment:
The existing model is to keep track of config in main.ts and pass values through to the Recorder class as args. I think this makes reconfiguring easier.

Reply:
Honestly, I hadn't even noticed that `cfg.get()` was exposed via an IPC channel. 🤦‍♀️ I will fix, of course :)

Reply:
Fixed to be in line with the existing way of passing configuration to the `Recorder`.